### PR TITLE
New: App main styles

### PR DESCRIPTION
**Pull request by Rimarika**

### CODE REVIEW

File: src/scss/_header.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.todo {
+  position: relative;
+  min-height: 100vh;
+  overflow: hidden;
```
Comment (grrido) [inline src/scss/_header.scss]: It is not a part of header, move this to another file

File: src/scss/_input-wrapper.scss [hunk 2]
```diff
@@ -0,0 +1,126 @@
+.todo {
```
Comment (grrido) [inline src/scss/_input-wrapper.scss]: Dont use it here

File: src/scss/_list-container.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.todo {
```
Comment (grrido) [inline src/scss/_list-container.scss]: Dont use it here

File: src/scss/_header.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        font-size: 1.5rem;
+        line-height: 26px;
+        text-align: center;
+      }
+    }
+  }
+}
+
+@media (min-width: 768px) {
+  .todo {
```
Comment (grrido) [inline src/scss/_header.scss]: Dont use it here - .todo

File: src/scss/_header.scss [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        display: block;
+        font-size: 1.5rem;
+        line-height: 26px;
+        text-align: center;
+      }
+    }
+  }
+}
+
+@media (min-width: 768px) {
```
Comment (grrido) [inline src/scss/_header.scss]: Put media inside, for example .title { some styles; @ media(...) }. And maybe create mixin or something for this media query to keep this 768px in one place

File: src/scss/_header.scss [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+      display: none;
+
+      &--active {
+        display: block;
+        font-size: 1.5rem;
+        line-height: 26px;
+        text-align: center;
+      }
+    }
+  }
```
Comment (grrido) [inline src/scss/_header.scss]: Put this all sub elements classes inside header class - .header { &__title {} ... }. And keep convention to not use .todo in rest files

File: src/scss/style.scss [hunk 7]
```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+@import 'reset';
+@import 'mixins';
+@import 'header';
+@import 'input-wrapper';
```
Comment (grrido) [inline src/scss/style.scss]: Input-wrapper? maybe just form?

File: src/scss/style.scss [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+@import 'list-container';
```
Comment (grrido) [inline src/scss/style.scss]: Just listing

File: src/scss/style.scss (hunk 7 above)
Comment (Rimarika) [reply src/scss/style.scss]: I think form-box is better

File: src/scss/_form-box.scss [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    .form-box__button {
+      position: absolute;
+      top: 0;
+      right: 0;
+      width: 30px;
+      height: 0;
+      margin: 0 5px;
+      cursor: pointer;
```
Comment (grrido) [inline src/scss/_form-box.scss]: Why .form-box is not replace by & and placed in form-box block?

File: src/scss/_header.scss [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+    .header__trigger-ico {
+      display: none;
+
+      &--active {
+        display: block;
+        font-size: 1.5rem;
+        line-height: 26px;
+        text-align: center;
+      }
+    }
```
Comment (grrido) [inline src/scss/_header.scss]: Why not &__trigger in .header block?

File: src/scss/_todo-box.scss [hunk 11]
```diff
@@ -0,0 +1,5 @@
+.todo {
+  position: relative;
+  min-height: 100vh;
+  overflow: hidden;
+}
```
Comment (grrido) [inline src/scss/_todo-box.scss]: Rename this file to _todo

**Review by grrido — APPROVED** [review]

**Pull request merged**